### ISSUE TITLE
Which are the actual language files in transifex?

### ISSUE THREAD

**Issue by jojo12** [jemproject/JEM-Project#1810]
I am not sure: the new language files go in transifex to JEM 2 or JEM 4?
For me it seems it goes in both. Is this right. 
Do we have to translate in both sections? Or:
Which one will be taken for the stable language files?

**Comment by mckillo** [comment]
I believe that the current version is only using JEM4.

**Comment by Heklaterriol** [comment]
The en-GB langauge files from the master branch are transferred to transifex/JEM 4 withing 24 hours after being modified.

I checked one of the new language strings, "COM_JEM_JANUARY" – it is both in JEM 2 and JEM 4.
So it seem that the language files are also transferreed to JEM 2.
As I have access to the German language files I checked the translation. There is one difference:
In JEM 2 it says `Translated by jojo12, 19 days ago via File upload`
In JEM 4 it says `Translated by hekla, 19 days ago via Editor`

In fact I translated the newest untranslated strings in JEM 4 into German some minutes after the transfer to transifex.
@jojo12: To understand the transifex mechanism: did you upload the German translations again or did you modify some of them.
